test/letters/containers/VeteranBenefitSummaryLetter.unit.spec.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+import { expect } from 'chai';
+
+import { VeteranBenefitSummaryLetter } from '../../../src/js/letters/containers/VeteranBenefitSummaryLetter';
+
+const defaultProps = {
+  benefitSummaryOptions: {
+    benefitInfo: {
+      awardEffectiveDate: '1965-01-01T05:00:00.000+00:00',
```

Comment:
ah yes I noticed this too, not sure what the fix for this is yet but we can address is a separate ticket